single_session/signals.py

```diff
@@ -15,7 +30,11 @@ def remove_other_sessions(sender, user, request, **kwargs):
     associated to that user.
     """
     # remove other sessions
-    if getattr(settings, "SINGLE_USER_SESSION", True):
+    session_validate_function = None
+    session_setting = getattr(settings, "SINGLE_USER_SESSION", True)
+    if isinstance(session_setting, str):
+        session_validate_function = import_string(getattr(settings, "SINGLE_USER_SESSION"))
```

Comment:
I wanted to do this check once when the package was imported (e.g. in line 13-23 in this PR), however this didn't work when overriding the setting on a case-by-case basis in the tests, so I do the import here each time instead. I'm open to suggestions on how we could improve this

Reply:
Yes, unfortunately I found out that Django's reload settings trigger does not really reinterpret the signals, apps, etc.

An "ugly" hack could be to listen to the settings change, and work with a non-local variable that sets the handler, although it is not very elegant.

Reply:
I had a look around at the django code, and it seems like they tend to perform an import in-line in the code wherever it's needed, e.g. 

https://github.com/django/django/blob/017fa23d3b0aee9142f531c2a0002fc86c82a54c/django/contrib/auth/hashers.py#L97
https://github.com/django/django/blob/017fa23d3b0aee9142f531c2a0002fc86c82a54c/django/contrib/messages/storage/__init__.py#L12

In that case, I wonder if this is good enough? It's certainly simpler than adding another settings change handler.